doc/en/user/source/installation/upgrade.rst

```diff
@@ -30,6 +30,29 @@ The general GeoServer upgrade process is as follows:
 Notes on upgrading specific versions
 ------------------------------------
 
+Log4J Upgrade (GeoServer 2.21 and newer)
```

Comment:
Not sure this is relevant here either.

Reply:
It is indeed off topic; I was going through the pages to see if workspace was mentioned. I will take this to a separate PR.